configurations/Blyth.json

```diff
@@ -5,20 +5,66 @@
             "Bus": 7,
             "Name": "Ambient 2 Temp",
             "Name1": "Relative Humidity",
+            "Thresholds": [
+                {
+                    "Direction": "greater than",
+                    "Index": 1,
+                    "Name": "Ambient Upper Critical",
```

Comment:
Looks to work correctly. Patched on `hwmontempsensor`, override Blyth.json with this version, and restarted EM then HwmonTempSensor and see the warning thresholds:
```
> mkdir -p /etc/entity-manager/configurations
# Copy Blyth.json to /etc/entity-manager/configurations

> rm /var/configuration/system.json && systemctl restart xyz.openbmc_project.EntityManager.service

> cat /var/configuration/system.json
{
    ...
        "Name": "Rainier 2U Chassis",
        "Probe": [
            "com.ibm.ipzvpd.VSBP({'IM': [80, 0, 16, 1]})",
            "OR",
            "com.ibm.ipzvpd.VSBP({'IM': [80, 0, 16, 3]})"
        ],
        "Type": "Chassis"
    },
    "1299366671": {
        "Exposes": [
            {
                "Address": 64,
                "Bus": 7,
                "Name": "Ambient 2 Temp",
                "Name1": "Relative Humidity",
                "Thresholds": [
                    {
                        "Direction": "greater than",
                        "Index": 1,
                        "Name": "Ambient Upper Critical",
                        "Severity": 0,
                        "Value": 100
                    },
                    {
                        "Direction": "less than",
                        "Index": 1,
                        "Name": "Ambient Lower Critical",
                        "Severity": 0,
                        "Value": 0
                    }
                ],
                "Type": "SI7020"
            },
            {
                "Address": 72,
                "Bus": 7,
                "Name": "Ambient 0 Temp",
                "PollRate": 1,
                "Thresholds": [
                    {
                        "Direction": "greater than",
                        "Name": "Ambient Upper Critical",
                        "Severity": 0,
                        "Value": 100
                    },
                    {
                        "Direction": "less than",
                        "Name": "Ambient Lower Critical",
                        "Severity": 0,
                        "Value": 0
                    }
                ],
                "Type": "TMP75"
            },
            {
                "Address": 118,
                "Bus": 7,
                "Name": "Ambient 1 Temp",
                "Name1": "Station Pressure",
                "Thresholds": [
                    {
                        "Direction": "greater than",
                        "Index": 1,
                        "Name": "Ambient Upper Critical",
                        "Severity": 0,
                        "Value": 100
                    },
                    {
                        "Direction": "less than",
                        "Index": 1,
                        "Name": "Ambient Lower Critical",
                        "Severity": 0,
                        "Value": 0
                    }
                ],
                "Type": "DPS310"
            },...

> systemctl restart xyz.openbmc_project.hwmontempsensor.service

> busctl introspect xyz.openbmc_project.HwmonTempSensor /xyz/openbmc_project/sensors/temperature/Ambient_0_Temp
NAME                                                  TYPE      SIGNATURE RESULT/VALUE                             FLAGS
...
xyz.openbmc_project.Sensor.Threshold.Warning          interface -         -                                        -
.WarningAlarmHigh                                     property  b         false                                    emits-change
.WarningAlarmLow                                      property  b         false                                    emits-change
.WarningHigh                                          property  d         100                                      emits-change writable
.WarningLow                                           property  d         0                                        emits-change writable
...

> busctl introspect xyz.openbmc_project.HwmonTempSensor /xyz/openbmc_project/sensors/temperature/Ambient_1_Temp
NAME                                                  TYPE      SIGNATURE RESULT/VALUE                             FLAGS
...
xyz.openbmc_project.Sensor.Threshold.Warning          interface -         -                                        -
.WarningAlarmHigh                                     property  b         false                                    emits-change
.WarningAlarmLow                                      property  b         false                                    emits-change
.WarningHigh                                          property  d         100                                      emits-change writable
.WarningLow                                           property  d         0                                        emits-change writable
...

> busctl introspect xyz.openbmc_project.HwmonTempSensor /xyz/openbmc_project/sensors/temperature/Ambient_2_Temp
NAME                                                  TYPE      SIGNATURE RESULT/VALUE                             FLAGS
...
xyz.openbmc_project.Sensor.Threshold.Warning          interface -         -                                        -
.WarningAlarmHigh                                     property  b         false                                    emits-change
.WarningAlarmLow                                      property  b         false                                    emits-change
.WarningHigh                                          property  d         100                                      emits-change writable
.WarningLow                                           property  d         0                                        emits-change writable
...
```

Reply:
fwiw, I thought adding "Ambient" to the names would help clarify the thresholds are in reference to the ambient sensor of the devices since I find it could be confusing to have `"Index": 1` actually refers to the sensor `Name` and not `Name1`.